src/utilities/indexedDB-helper.js

```diff
@@ -45,8 +51,32 @@ export const saveSelections = async (newSelections) => {
         selectionList: JSON.parse(JSON.stringify(newSelections)),
       };
 
-      const trans = db.transaction(TABLE_NAME, "readwrite");
-      const store = trans.objectStore(TABLE_NAME);
+      const trans = db.transaction(SELECTION_TABLE_NAME, "readwrite");
+      const store = trans.objectStore(SELECTION_TABLE_NAME);
+
+      console.log("ADD TO INDEXEDDB", note);
+      const result = await store.put(note);
+
+      resolve(result);
+    } catch (error) {
+      console.error(`There was an error adding to drafts, ${error}`);
+      reject(error);
+    }
+  });
+};
+
+// Save API Data to indexedDB api_data table
+export const saveData = async (newSelections) => {
+  const db = await getDb();
+  return new Promise(async (resolve, reject) => {
+    try {
+      const note = {
+        selectionId: DATA_SESSION_ID,
+        selectionList: JSON.parse(JSON.stringify(newSelections)),
+      };
```

Comment:
the 'note' here should be `{dataId: DATA_SESSION_ID, dataList: JSON.parse(JSON.stringify(newData))`